pkg/asset/manifests/ingress.go

```diff
@@ -108,13 +109,24 @@ func (ing *Ingress) generateClusterConfig(config *types.InstallConfig) ([]byte,
 			Name: "cluster",
 			// not namespaced
 		},
-		Spec: configv1.IngressSpec{
-			Domain: fmt.Sprintf("apps.%s", config.ClusterDomain()),
-		},
+		Spec: configv1.IngressSpec{},
 		Status: configv1.IngressStatus{
 			DefaultPlacement: defaultPlacement,
 		},
 	}
+	obj.Spec.Domain = fmt.Sprintf("apps.%s", config.ClusterDomain())
```

Comment:
Are these changes needed? What's wrong with the way `Spec.Domain` was being set?